pkg/config/sync.go

```diff
@@ -384,6 +384,15 @@ func mergeConfig(from, to *controller.OperatorConfiguration) {
 			}
 			to.Workspace.DefaultContainerResources = mergeResources(from.Workspace.DefaultContainerResources, to.Workspace.DefaultContainerResources)
 		}
+
```

Comment:
Changes also need to be made to [`GetCurrentConfigString()`](https://github.com/devfile/devworkspace-operator/blob/main/pkg/config/sync.go#L487) to either give the list of the config's podAnnotations, or indicate that they are being set.

Reply:
updated